repo/packages/A/avi/2/resource.json

```diff
@@ -0,0 +1,23 @@
+{
+  "assets": {
+    "container": {
+      "docker": {
+        "controller": "avinetworks/controller",
+        "serviceengine": "avinetworks/se"
```

Comment:
these docker images should be a locked to a specific version... without specifying it assumes ":latest" which allows potentially unstable changes.